docs/shortcuts.html

```diff
@@ -1,664 +1,837 @@
-              <td> View the Task Manager </td>
-              <td> <span class="kbd">Shift</span> + <span class="kbd">ESC</span> </td>
-              <td> <span class="kbd">Shift</span> + <span class="kbd">ESC</span> </td>
-            </tr>
-            <tr>
-              <td> Next page in a tabs browsing history </td>
-              <td> <span class="kbd">Alt</span> + <span class="kbd">Right</span> </td>
-              <td> <span class="kbd">Alt</span> + <span class="kbd">Right</span> </td>
-            </tr>
-            <tr>
-              <td> Previous page in a tabs browsing history </td>
-              <td> <span class="kbd">Backspace</span>, <span class="kbd">Alt</span> + <span class="kbd">Left</span> </td>
-              <td> <span class="kbd">Backspace</span>, <span class="kbd">Alt</span> + <span class="kbd">Left</span> </td>
-             </tr>
-             <tr>
-               <td> Highlight content in the web address area </td>
-               <td> <span class="kbd">F6</span>, <span class="kbd">Ctrl</span> + <span class="kbd">L</span>, <span class="kbd">Alt</span> + <span class="kbd">D</span> </td>
-               <td> <span class="kbd">Cmd</span> + <span class="kbd">L</span>, <span class="kbd">Alt</span> + <span class="kbd">D</span> </td>
-             </tr>
-             <tr>
-               <td> Places a ? in the address bar for performing a keyword <br />
-                  search using your default search engine </td>
-               <td> <span class="kbd">Ctrl</span> + <span class="kbd">K</span>, <span class="kbd">Ctrl</span> + <span class="kbd">E</span> </td>
-               <td> <span class="kbd">Cmd</span> + <span class="kbd">K</span>, <span class="kbd">Cmd</span> + <span class="kbd">E</span> </td>
-             </tr>
-          </tbody>
-        </table>
+  <h2 id="other-shortcuts"> Other Chrome Shortcuts </h2>
+
+  <p>Here are some additional Chrome shortcuts which are useful for general use within the browser not specific to the DevTools. <a href="http://goo.gl/PsTNm">View all Chrome shortcuts</a> for Windows, Mac, and Linux. </p>
+
+  <table>
+    <tbody>
+      <tr>
+        <th></th>
+        <th> Windows / Linux </th>
+        <th> Mac </th>
+      </tr>
+      <tr>
+        <td> Find next </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">G</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">G</span> </td>
+      </tr>
+      <tr>
+        <td> Find previous </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Shift</span> + <span class="kbd">G</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Shift</span> + <span class="kbd">G</span> </td>
+      </tr>
+      <tr>
+        <td> Open a new window in Incognito mode </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Shift</span> + <span class="kbd">N</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Shift</span> + <span class
```

Comment:
Doesn't seem to work - it looks like task manager doesn't have a defined keyboard shortcut:

![screen shot 2015-01-12 at 22 21 24](https://cloud.githubusercontent.com/assets/985504/5711275/67bc639e-9aa9-11e4-991a-7f32ef488f9f.png)